scripts/add_existing_baseyear.py

```diff
@@ -639,29 +638,43 @@ def add_heating_capacities_installed_before_baseyear(
     )
 
     if options["heating"]:
-        time_dep_hp_cop = options["time_dep_hp_cop"]
-        ashp_cop = (
-            xr.open_dataarray(snakemake.input.cop_air_total)
-            .to_pandas()
-            .reindex(index=n.snapshots)
-        )
-        gshp_cop = (
-            xr.open_dataarray(snakemake.input.cop_soil_total)
-            .to_pandas()
-            .reindex(index=n.snapshots)
-        )
-        default_lifetime = snakemake.params.existing_capacities[
-            "default_heating_lifetime"
-        ]
+
         add_heating_capacities_installed_before_baseyear(
-            n,
-            baseyear,
-            grouping_years_heat,
-            ashp_cop,
-            gshp_cop,
-            time_dep_hp_cop,
-            costs,
-            default_lifetime,
+            n=n,
```

Comment:
style suggestion: to make code less nested, perhaps created variables `cop` and `existing_heating` before function call and just pass object?

Reply:
Hm, can do but that would create unnecessary memory load, wouldn't it?

Reply:
Also fine to leave as is, but I don't think it would create a memory overhead since no copy is made.